src/analog.jl

```diff
@@ -183,7 +183,7 @@ function read(t::AITask, num_samples_per_chan::Integer = -1, precision::DataType
         Ref(num_samples_per_chan_read,1),
         reinterpret(Ptr{Bool32},C_NULL)) )
     data = data[1:num_samples_per_chan_read[1]*num_channels]
-    num_channels==1 ? data : reshape(data, (div(length(data),num_channels), convert(Int64,num_channels)))
+    return num_channels==1 ? data : reshape(data, (div(length(data),num_channels), convert(Int64,num_channels)))
```

Comment:
ditto--  adding a return statement here does not have anything to do with updating to the latest version of the drivers.  generally it's not good to make random changes like this.